src/util-logopenfile.h

```diff
@@ -114,6 +114,9 @@ typedef struct LogFileCtx_ {
     /* flag to avoid multiple threads printing the same stats */
     uint8_t flags;
 
+    /* flag to store the options */
+    uint8_t options_flags;
```

Comment:
I don't think this is the proper place to add these options. They are about the JSON output, so they should move into something JSON specific.